server.go

```diff
@@ -376,8 +379,10 @@ func newServer(cfg *Config, listenAddrs []net.Addr, chanDB *channeldb.DB,
 	copy(serializedPubKey[:], nodeKeyECDH.PubKey().SerializeCompressed())
 
 	// Initialize the sphinx router, placing it's persistent replay log in
-	// the same directory as the channel graph database.
-	sharedSecretPath := filepath.Join(cfg.localDatabaseDir(), "sphinxreplay.db")
+	// the same directory as the channel graph database. We don't need to
+	// replicate this data, so we'll store it locally.
+	graphDir := localChanDB.Path()
+	sharedSecretPath := filepath.Join(graphDir, "sphinxreplay.db")
```

Comment:
nit: now that you're touching this code, consider introducing a constant for the db filename.